lib/Conch/Controller/User.pm

```diff
@@ -279,7 +279,10 @@ sub create ($c) {
 
 	# we don't use lookup_by_* because they only search active users.
 	if (my $user = $c->db_user_accounts->search({
-			-or => { name => $name, email => $email }
```

Comment:
For existing users with multiple accounts based on case sensitivity, this will return multiple records.